src/Libraries/Microsoft.Extensions.AI.Abstractions/ChatCompletion/ChatMessage.cs

```diff
@@ -10,6 +10,7 @@
 namespace Microsoft.Extensions.AI;
 
 /// <summary>Represents a chat message used by an <see cref="IChatClient" />.</summary>
+/// <related type="Article" href="/dotnet/ai/quickstarts/build-chat-app">Build an AI chat app with .NET</related>
```

Comment:
```suggestion
/// <related type="Article" href="/dotnet/ai/quickstarts/build-chat-app">Build an AI chat app with .NET.</related>
```

Reply:
We don't generally put periods/full stops on See also links, so I'm going to resolve this. But let me know if you feel strongly.

Reply:
This repo uses some analyzers that report a warning when the full stop is missing for a comment. That's unfortunately causing the CI build to fail.

Ideally, the analyzer that reports this warning would ignore missing full stops in `<related>` tags. But absent that, the easiest fix is probably to add the full stops. Otherwise, you would need to add `#pragma` to suppress these warnings individually (or suppress them globally in the corresponding projects which could mean also ignoring legit missing full stop bugs in other tags in the future...).

Reply:
Okay, I'll add them. As any FYI, these become bullets in a list [like this](https://learn.microsoft.com/en-us/dotnet/api/system.linq.parallelquery-1.getenumerator?view=net-9.0#see-also).